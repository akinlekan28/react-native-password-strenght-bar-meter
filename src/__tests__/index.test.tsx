import React from 'react';
import renderer from 'react-test-renderer';
import PasswordStrengthMeterBar from '../index';

jest.useFakeTimers();

test('renders correctly', async () => {
  const tree = renderer
    .create(<PasswordStrengthMeterBar password="testpassword" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
