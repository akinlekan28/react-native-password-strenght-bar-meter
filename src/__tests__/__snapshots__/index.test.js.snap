// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 5,
    }
  }
>
  <View />
</View>
`;
